CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.18] - 2022-10-02
+
+### Added
+
+- Support for enabling flexible enpointing via `max_delay_mode` in transcription config.
```

Comment:
Isn't flexible endpointing going to be the default? In which case this is confusing - you don't need to use `max_delay_mode` to get flexible endpointing. I'd just say add support for choosing mode of operation for `max_delay`.

Reply:
Correct observation - changed